crates/libcontainer/src/container/tenant_builder.rs

```diff
@@ -327,8 +327,16 @@ impl TenantContainerBuilder {
 
         let init_process = procfs::process::Process::new(container_pid.as_raw())?;
         let ns = self.get_namespaces(init_process.namespaces()?.0)?;
-        let linux = LinuxBuilder::default().namespaces(ns).build()?;
 
+        let linux = spec.linux().as_ref().unwrap();
+        let linux = if linux.cgroups_path().is_some() {
+            LinuxBuilder::default()
+                .namespaces(ns)
+                .cgroups_path(linux.cgroups_path().as_ref().unwrap().clone())
+                .build()?
+        } else {
+            LinuxBuilder::default().namespaces(ns).build()?
+        };
```

Comment:
```rust
        let linux_spec = spec.linux().as_ref().unwrap();
        let linux = if linux_spec.cgroups_path().is_some() {
            LinuxBuilder::default()
                .namespaces(ns)
                .cgroups_path(linux_spec.cgroups_path().as_ref().unwrap().clone())
                .build()?
        } else {
            LinuxBuilder::default().namespaces(ns).build()?
        };
```
Maybe it would be better if we rename this var name.

Reply:
yep, will do that as well :+1: